apps/api/src/app/environments-v1/usecases/output-renderers/maily-to-liquid/maily.types.ts

```diff
@@ -1,5 +1,7 @@
 export enum MailyContentTypeEnum {
   VARIABLE = 'variable',
+  REPEAT = 'repeat',
+  // IS DEPRECATED, but kept for backwards compatibility
```

Comment:
for a bit better DX.

```suggestion
  /**
 * Legacy enum value maintained for backwards compatibility
   * @deprecated
   */
   ```